mangaki/mangaki/utils/common.py

```diff
@@ -47,5 +48,14 @@ def get_backup_filename(self):
     def compute_rmse(self, y_pred, y_true):
         return mean_squared_error(y_true, y_pred) ** 0.5
 
+    def all_errors(self, X_train, X_test, y_train, y_test):
+        self.fit(X_train, y_train)
```

Comment:
I'd prefer that `all_errors` only predict and compute errors and should be called `print_all_errors` or should return errors as values rather than printing them.